lib/front-end-builds-notifier.js

```diff
@@ -1,5 +1,4 @@
 var CoreObject = require('core-object');
-var Promise = require('rsvp').Promise;
```

Comment:
if this is removed, where is this being imported?

Reply:
It'll use a native `Promise` object—all the versions of Node we use have this well and truly now.